examples/treegrid/treegrid-1.html

```diff
@@ -0,0 +1,567 @@
+      <li><a href="../../#treegrid">Treegrid Design Pattern</a></li>
+    </ul>
+</nav>
+  <main>
+  <h1>Treegrid Email Inbox Example</h1>
+
+  <p>
+    <strong>NOTE:</strong> This example page is not yet complete.
+    Development of this example is tracked by
+    <a href="https://github.com/w3c/aria-practices/issues/132">issue 132.</a>
+  </p>
+  <p>
+    The following example demonstrates how the
+    <a href="../../#treegrid">treegrid design pattern (TBD)</a>
+    can be used to make an interactive tree that enables users to navigate the hierarchical structure of email conversations
+    and also provides the ability to navigate to elements that describe each email, such as subject and sender.
+  </p>
+  <p>
+    This same example can be used in three ways.
+  </p>
+  <ul>
+    <li id="option-cell-focus-allow"><a href="./treegrid-1.html">Rows are focused first, but cells can be focused</a>:<br>Useful when we want to present the treegrid to act primarily like a tree.</li>
+    <li id="option-cell-focus-start"><a href="./treegrid-1.html?cell=start">Cells are focused first, but rows can also be focused</a>:<br>Useful when we want to present the treegrid to act primarily like a grid.</li>
+    <li id="option-cell-focus-force"><a href="./treegrid-1.html?cell=force">Only cells can be focused</a>:<br>Useful when we we are primarily a grid, and we want to simplify the user experience by removing row navigation options.</li>
+  </ul>
+  <p>
+    Note: no row-only version is provided. A row-only navigation version could be useful when the columns are guaranteed to fit horizontally (no horizontal scrolling necessary), the columns are merely for attractive presentation, as opposed to addressing any need to visit them individually. However, in this case the author can simply follow the tree pattern and present the child treeitem data in columns.
+  </p>
+
+  <section>
+    <h2 id="ex_label">Example</h2>
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <!--
+      Note the ID of the following div that contains the example HTML is used as a parameter for the sourceCode.add() function.
+      The sourceCode functions in the examples/js/examples.js render the HTML source to show it to the reader of the example page.
+      If you change the ID of this div, be sure to update the parameters of the sourceCode.add() function call, which is made following the div with id="sc1" where the HTML is render.
+      The div for the rendered HTML source is in the last section of the page.
+    -->
+    <div id="ex1">
+      <table id="treegrid" role="treegrid" aria-label="ARIA treegrid e-mail inbox example">
+        <colgroup>
+          <col id="treegrid-col1">
+          <col id="treegrid-col2">
+          <col id="treegrid-col3">
+        </colgroup>
+        <thead>
+          <tr>
+            <th role="columnheader">Subject</th>
+            <th role="columnheader">Summary</th>
```

Comment:
@aleventhal, @jongund, why is role row added here and on subsequent tr elements? This is not necessary and not recommended to duplicate native semantics. Can we safely remove without breaking any selectors or js?

Reply:
Good question. I would guess that it makes no difference and can be removed, but I think it should be tested with at least one screen reader to be completely sure.

FWIW, I find it easier to understand the examples when the roles are explicitly stated.  When we remove "row" from the markup, should we also make sure it's clear in the docs that <tr> can be replaced with something like <div role="row"> ?